packages/react-native-editor/ios/.ruby-version

```diff
@@ -1 +1 @@
-2.7.5
+2.7.4
```

Comment:
That is the version supported by our CI. We could of course install a new one at runtime, but that adds 4 to 5 minutes to the build.

<img width="1150" alt="image" src="https://github.com/WordPress/gutenberg/assets/1218433/166117a9-e25d-4bf9-8dac-2a8265185a2b">


Worth noting that version 2.7.4 and 2.7.5 both are past support. We ought to upgrade to version 3.x, but that's something we might want to do in a dedicated PR.